widgetssdk/build.gradle

```diff
@@ -37,6 +37,10 @@ android {
                 'WrongViewIdFormat',
                 'HardcodedText'
     }
+
+    buildFeatures {
+        viewBinding true
+    }
```

Comment:
Would this require the integrators to also add this to their project?

Reply:
If I'm not mistaken no, it generates binding classes for views at compile time

Reply:
I did an investigation. It doesn't require the integrators to use the view binding in their projects.